packages/core/src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -686,4 +705,28 @@ export class AbstractQueryInterfaceTypeScript {
   ): Promise<void> {
     await this.sequelize.queryRaw(this.queryGenerator.getToggleForeignKeyChecksQuery(enable), options);
   }
+
+  /**
+   * Delete records from a table
+   *
+   * @param tableName
+   * @param options
+   */
+  async delete(tableName: TableNameOrModel, options: QiDeleteOptions): Promise<number> {
+    const sql = this.queryGenerator.deleteQuery(tableName, options);
+    // unlike bind, replacements are handled by QueryGenerator, not QueryRaw
+    delete options.replacements;
```

Comment:
we should not mutate parameters. `options` used to be cloned but now isn't

Instead of cloning, you can set `replacements` to `undefined` in `queryRaw`

Reply:
Done